libs/backward.hpp

```diff
@@ -1918,7 +1918,7 @@ class cfile_streambuf: public std::streambuf {
     return traits_type::eof();
   }
 
-  std::streamsize xsputn(const char_type* s, std::streamsize count) {
+  std::streamsize xsputn(const char_type* s, std::size_t count) {
```

Comment:
Return value as well?

Reply:
fwrite returns size_t, so it should be that.

Reply:
Yep, you're right!